content/tutorials/guides/how-to-collect-product-variations.md

```diff
@@ -0,0 +1,277 @@
+
+
+Here are quick definitions:
+
+**A product model gathers products with many similarities but a few differences - these differences are called “variations” and are defined by an associated family variant.**
+
+*Example: T-shirts with the same material (”wool”), brand (”a brand”), and weight (”16 grams”); but with different sizes and colors.*
+
+**A product variant is associated with a product model. It contains the common attribute values of its product model plus values for variations.**
+
+*Example: A t-shirt with material = ”whool”, brand = ”a brand”, weight = ”16 grams”, size = “XL”, and color = “blue*”.
+
+And because a relation schema is worth a thousand words:
+
+![scheme_model_and_variant](../../img/tutorials/how-to-collect-product-variations/scheme_model_and_variant.svg)
+
+::: tips
+Get the big picture <a href="/getting-started/synchronize-pim-products-6x/step-0.html" target="_blank" rel="noopener noreferrer">here</a>.
+:::
+
+
+## Collect product variations
+
+### Workflow
+
+![schema_product_models](../../img/tutorials/how-to-collect-product-variations/schema_product_models.svg)
+
+In this tutorial, we will introduce you to the two use cases you may encounter for retrieving products with variations.
+
+- **Use case 1**: Collect all product variation information, just as they are in the PIM (up to 2 possible levels)
+- **Use case 2**: Collect all product variation information on 1 level only)
+
+::: info
+We noticed that many e-commerce solutions understand product variation on only one level. This means that for Akeneo a special recollection of the variations must be done to have them all on the same level.
+:::
+
+### 0 - Initialization
+
+```php [activate:PHP]
+
+$pimUrl = 'https://url-of-your-pim.com';
+$appToken = 'your_app_token'; // Token provided during oAuth steps
+
+// If you haven't done it yet, please follow the Guzzle official documentation for installing the client 
+// https://docs.guzzlephp.org/en/stable/overview.html#installation
+
+// Set your client for querying Akeneo API as follows
+function getApiClient(): GuzzleHttp\Client
+{
+    $pimUrl = '<PIM_URL>';
+    $appToken = '<APP_TOKEN>'; // Token provided during oauth steps
+
+    return new GuzzleHttp\Client([
+        'base_uri' => $pimUrl,
+        'headers' => ['Authorization' => 'Bearer ' . $appToken],
+    ]);
+}
+
+```
+
+### Use case 1: Collect product variation information - all levels
+
+#### 1. Collect product models
+##### 1.1 You are following the App workflow?
+
+In the guided tutorial [**"How to get families and attributes"**](/tutorials/how-to-get-families-and-attributes.html), we have stored a family_code_list. It’s time to use it!
+
+```php [activate:PHP]
+
+function getProductModels(): array
+{
+    $client = getApiClient();
+
+    $maxProductsPerPage = 100;
+    $maxFamiliesPerQuery = 3;
+    $scope = 'ecommerce';
+
+    // Get family $codes and locales from storage
+    $familyCodes = getFamilyCodes();
+    $locales = getLocales('fr');
```

Comment:
You should get your locale from your storage

Reply:
that's exactly what I do